frontend/src/pages/videos/VideoRatings.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useLocation, useHistory, Link } from 'react-router-dom';
+import { Box, Button, Divider } from '@material-ui/core';
+
+import type {
+  ContributorRating,
+  PaginatedContributorRatingList,
+} from 'src/services/openapi';
+import Pagination from 'src/components/Pagination';
+import VideoList from 'src/features/videos/VideoList';
+import { UsersService } from 'src/services/openapi';
+import { ContentBox, LoaderWrapper } from 'src/components';
+import {
+  PublicStatusAction,
+  RatingsContext,
+} from 'src/features/videos/PublicStatusAction';
+import RatingsFilter from 'src/features/ratings/RatingsFilter';
+import { scrollToTop } from 'src/utils/ui';
+
+const NoRatingMessage = ({ hasFilter }: { hasFilter: boolean }) => (
+  <>
+    <Divider />
+    {hasFilter ? (
+      <Box my={2}>{'No video in this list.'}</Box>
```

Comment:
I would rather use a sentence formulated like the one used in Recommendation page : `No video corresponds to your filter`

Reply:
Fixed in 55d901f